sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -172,7 +191,7 @@ export class EducationProgramController {
       programId,
       userToken.authorizations.institutionId,
     );
-    return {
+    const programDetails: SubsetEducationProgramDto = {
```

Comment:
Please take a look at the comments in the body of this method. IMO it makes more sense return only the new data without further processing.